src/workers/parser/utils/contains.js

```diff
@@ -3,14 +3,26 @@
 import type { AstLocation, AstPosition } from "../types";
 import type { Node } from "babel-traverse";
 
-export function containsPosition(a: AstLocation, b: AstPosition) {
-  const startsBefore =
-    a.start.line < b.line ||
-    (a.start.line === b.line && a.start.column <= b.column);
-  const endsAfter =
-    a.end.line > b.line || (a.end.line === b.line && a.end.column >= b.column);
+function startsBefore(a: AstLocation, b: AstPosition) {
+  let startsBefore = a.start.line < b.line;
+  if (a.start.line === b.line) {
+    startsBefore =
+      a.start.column >= 0 && b.column >= 0 ? a.start.column <= b.column : true;
```

Comment:
The getOutOfScopeLocations.spec.js test shows "0"[1] matters so `>= 0` is required.

[1] https://github.com/devtools-html/debugger.html/blob/c5177581a8af4b797f9aa56f82214216de0176c6/src/workers/parser/tests/getOutOfScopeLocations.spec.js#L29